doc/source/serve/package-ref.md

```diff
@@ -47,169 +47,11 @@
 .. autofunction:: ray.serve.batch(max_batch_size=10, batch_wait_timeout_s=0.0)
 ```
 
-## Serve REST API
+## Extended API
```

Comment:
Are there any suggestions for another name for this section? "Non-Python API" doesn't sound as nice, "Other API" is possible although generic. cc: @shrekris-anyscale @simon-mo @edoakes

Reply:
Operational APIs
Please checkout the REST API and CLI that are useful for running, debugging, inspecting, and deploying the application in production.